base/compiler/tfuncs.jl

```diff
@@ -2084,12 +2084,13 @@ function array_type_undefable(@nospecialize(arytype))
     return true
 end
 
-@nospecs function memoryset_typecheck(memtype, elemtype)
+@nospecs function memoryset_typecheck(𝕃::AbstractLattice, memtype, elemtype)
     # Check that we can determine the element type
     isa(memtype, DataType) || return false
     elemtype_expected = memoryref_elemtype(memtype)
     elemtype_expected === Union{} && return false
     # Check that the element type is compatible with the element we're assigning
+    ⊑ = Core.Compiler.:⊑(𝕃)
```

Comment:
Aren't we already in Core.Compiler here?

Reply:
This is a hack to use the same symbol as a binary operator in the local context.

Reply:
A little ugly, but I guess I don't care.

Reply:
Should be just `Compiler.⊑` then, since `Core.Compiler` might be a different module from this

Reply:
tfuncs.jl is loaded only within `Core.Compiler`, but let's make this clean in another PR.

Reply:
I often load other Compilers that aren't the one in Core